conduct.md

```diff
@@ -132,7 +132,13 @@ TODO: Write an introductory paragraph.
 
 ### Infractions
 
-TODO: Write an introductory paragraph.
+Infractions are violations of the class conduct guide as specified by the unacceptable conduct sections.
+
+When a violation is committed, the violation is added to a point total and a penalty is given according to the total violation count received by the student.  The point total is counted throughout the semester and is not removed until the class is over for the semester.
```

Comment:
Since each violation corresponds to one point, might this infraction policy be easier to understand if the point system was removed altogether? For instance, anywhere it refers to a student receiving a point could be reworded to say that the student has committed a violation.